Charts/values.yaml

```
@@ -0,0 +1,10 @@
+replicaCount: 1
+
+image:
+  repository: ghcr.io/spectrocloud/librarium
+  tag: "nightly"
+  pullPolicy: Always
+
+service:
+  type: ClusterIP
+  port: 8000
```

```suggestion
  port: 8080
```
Can we make this `8080`, just so that it matches the expected for we documented for the manual steps.
https://docs.spectrocloud.com/vertex/install-palette-vertex/airgap/offline-docs/